### PR TITLE
Feature/deploy schemas 3

### PR DESCRIPTION
**Pull request by rlaferla**
Problem
=======
Need method to register schemas.

Closes: #3 

Solution
========
App to register one or more schemas with Frequency blockchain.

### CODE REVIEW

File: deploy.test.ts [hunk 1]
```diff
@@ -1,5 +1,9 @@
+import { describe, expect } from "@jest/globals";
 import { deploy } from "./deploy";
 
+//jest.setTimeout(5000);
+jest.useFakeTimers();
+
 describe("Mock Deploy", () => {
   it("does not fail", async () => {
     await expect(() => {
```
Comment (wilwade) [inline deploy.test.ts:9]: I was looking at this closer as the test still passes when I am not running a local frequency.

I think we need the test like this:

```suggestion
    await expect(deploy()).resolves.not.toThrow();
```

That then correctly errors out.

It however might require a few changes to fix. Thoughts on removing the test instead?

**Review by wilwade — CHANGES_REQUESTED** [review]
Few small cleanups. Deploys nicely

File: deploy.test.ts (hunk 1 above)
Comment (rlaferla) [reply deploy.test.ts]: Removing the test.

**Review by wilwade — APPROVED** [review]
🚢

**Pull request merged**